client/tests/signing/mod.rs

```diff
@@ -0,0 +1,5 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+mod alias;
```

Comment:
Would like to have it merged rather sooner because the changes to the others tests can be conflicting and I also want to add quite a few more tests which would make the PR way bigger

Reply:
Fair enough